umm-spec-lib/src/cmr/umm_spec/field_update.clj

```diff
@@ -77,9 +77,12 @@
 
 (defmethod apply-umm-list-update :add-to-existing
   [update-type umm update-field update-value find-value]
-  (if (sequential? update-value)
-    (update-in umm update-field #(concat % update-value))
-    (update-in umm update-field #(conj % update-value))))
+  (let [umm (if (sequential? update-value)
```

Comment:
Nitpick: you could do this function as one thread if you wanted to:

```
(as-> umm umm
  (if (sequential? update-value)
    (update-in umm update-field #(concat % update-value))
    (update-in umm update-field #(conj % update-value)))
  (util/update-in-each umm update-field util/remove-nil-keys)
  (update-in umm update-field distinct))
```

Both work, but this is a little clearer to me
https://clojuredocs.org/clojure.core/as-%3E